docker-app/qfieldcloud/core/exceptions.py

```diff
@@ -169,6 +162,7 @@ class ProjectAlreadyExistsError(QFieldCloudException):
     status_code = status.HTTP_400_BAD_REQUEST
 
 
+@deprecated("moved to subscription")
```

Comment:
Do we really need to deprecate them, can't we just delete them?

Reply:
oh sure, only keep  ReachedMaxOrganizationMembersError